PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/selection/PointMutationBubbleDescriptionController.java

```diff
@@ -0,0 +1,94 @@
+package nl.tudelft.pl2016gr2.gui.view.selection;
+
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListView;
+import nl.tudelft.pl2016gr2.model.GenomeMap;
+import nl.tudelft.pl2016gr2.model.graph.nodes.GraphNode;
+import nl.tudelft.pl2016gr2.model.graph.nodes.PointMutationBubble;
+import nl.tudelft.pl2016gr2.model.graph.nodes.SequenceNode;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ResourceBundle;
+import java.util.stream.Collectors;
+
+/**
+ * This controller controls the selection View for a {@link PointMutationBubble}.
+ *
+ * <p>
+ * It shows two lists containing the genome names for each of the base
+ * mutation they belong to.
+ * </p>
+ */
+public class PointMutationBubbleDescriptionController implements Initializable {
+
+  @FXML
+  private Label labelGenomesOne;
+  @FXML
+  private Label labelGenomesTwo;
+
+  @FXML
+  private ListView<String> listViewGenomesOne;
+
+  @FXML
+  private ListView<String> listViewGenomesTwo;
+
+  @Override
+  public void initialize(URL url, ResourceBundle resourceBundle) {
+  }
+
+  /**
+   * Call this to set-up this view.
+   *
+   * <p>
+   * Fills in the data of this view. Please refer to
+   * {@link PointMutationBubbleDescriptionController}
+   * for more information about this view.
+   * </p>
+   *
+   * @param bubble the {@link PointMutationBubble} that holds the data.
+   */
+  public void setup(PointMutationBubble bubble) {
+    if (bubble.getChildren().size() == 2) {
+      Iterator<GraphNode>  it = bubble.getChildren().iterator();
+      GraphNode childOne = it.next();
+      GraphNode childTwo = it.next();
+
+      if (childOne.getClass() == SequenceNode.class && childTwo.getClass() == SequenceNode.class) {
```

Comment:
It should be true that they are non-bubbles, but that would be better to check with `hasChildren`. If we have another root-level node implementation, this breaks (We nearly had a ConvergentEvolutionNode, so it can happen)